### PR TITLE
migration

### PR DESCRIPTION
**Pull request by banadiga**

### CODE REVIEW

File: src/main/resources/db/migration/V2.2__emails_incrice_langth.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+-- alter CAMPAIGNS
```
Comment (iverhun) [inline src/main/resources/db/migration/V2.2__emails_incrice_langth.sql]: incrice?? Or increase?

Comment (iverhun) [reply src/main/resources/db/migration/V2.2__emails_incrice_langth.sql]: langth??? OR length?

File: src/main/resources/db/migration/V2.1__tickets_url_nulable.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+-- alter EVENTS
+
+ALTER TABLE EVENTS ADD COLUMN NEW_TICKETS_URL VARCHAR(255);
+UPDATE EVENTS SET NEW_TICKETS_URL=TICKETS_URL;
+ALTER TABLE EVENTS DROP COLUMN TICKETS_URL;
+RENAME COLUMN EVENTS.NEW_TICKETS_URL TO TICKETS_URL;
```
Comment (iverhun) [inline src/main/resources/db/migration/V2.1__tickets_url_nulable.sql]: Why not just do: 
```
ALTER TABLE EVENTS MODIFY TICKETS_URL VARCHAR(255);
```

https://stackoverflow.com/questions/212939/how-do-i-modify-a-mysql-column-to-allow-null

File: src/main/resources/db/migration/V2.2__emails_incrice_langth.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+ALTER TABLE CAMPAIGNS ADD COLUMN NEW_BODY VARCHAR(32672) NOT NULL default '';
+UPDATE CAMPAIGNS SET NEW_BODY=BODY;
+ALTER TABLE CAMPAIGNS DROP COLUMN BODY;
+RENAME COLUMN CAMPAIGNS.NEW_BODY TO BODY;
```
Comment (iverhun) [inline src/main/resources/db/migration/V2.2__emails_incrice_langth.sql]: Why not just do:
```
ALTER TABLE CAMPAIGNS MODIFY BODY VARCHAR(32672);
```

https://stackoverflow.com/questions/1279568/how-can-i-modify-the-size-of-column-in-a-mysql-table

File: src/main/resources/db/migration/V2.1__tickets_url_nulable.sql (hunk 2 above)
Comment (banadiga) [reply src/main/resources/db/migration/V2.1__tickets_url_nulable.sql]: Syntax error: Encountered "MODIFY" at line 1, column 20.

`MODIFY` is not supported

**Pull request merged**